jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/DefaultConfigFilesFacadeTest.kt

```diff
@@ -310,6 +310,215 @@ class DefaultConfigFilesFacadeTest {
         )
     }
 
+    @Test
+    fun `delete session from config on sign out - only sso-session`() {
+        val baseFolder = folderRule.newFolder()
+        val config = Paths.get(baseFolder.absolutePath, ".aws", "config")
+        config.createParentDirectories()
+        config.writeText(
+            """
+            [sso-session precedingabc]
+            key1=value1
+            key2=value2
+            [profile precedingabc-1-a]
+            key1=value1
+            key2=value2
+            [sso-session abc]
+            key1=value1
+            key2=value2
+            """.trimIndent()
+        )
+        val creds = Paths.get(baseFolder.absolutePath, ".aws", "credentials")
+        val sut = DefaultConfigFilesFacade(configPath = config, credentialsPath = creds)
+        sut.deleteSsoConnectionFromConfig("abc")
+        assertThat(config).hasContent(
+            """
+            [sso-session precedingabc]
+            key1=value1
+            key2=value2
+            [profile precedingabc-1-a]
+            key1=value1
+            key2=value2
+            """.trimIndent()
+        )
+    }
+
+    @Test
+    fun `delete session from config on sign out`() {
+        val baseFolder = folderRule.newFolder()
+        val config = Paths.get(baseFolder.absolutePath, ".aws", "config")
+        config.createParentDirectories()
+        config.writeText(
+            """
+            [sso-session precedingabc]
+            key1=value1
+            key2=value2
+            [profile precedingabc-1-a]
+            key1=value1
+            key2=value2
+            [sso-session abc]
+            key1=value1
+            key2=value2
+            [profile abc-1-a]
+            sso_session=abc
+            key2=value2
+            [sso-session ac]
```

Comment:
maybe pick better names because it's hard to figure out the output when everything is a variation of abc123